custom-connectors/HR Cloud/apiDefinition.swagger.json

```diff
@@ -0,0 +1,962 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HR Cloud",
+    "description": "HR Cloud is a powerful and globally recognized HRMS software for the entire employee lifecycle.\nUser this connector to easily send employee information between HR Cloud and other systems you use.",
+    "version": "1.0",
+	"contact":{
+	"name": "HR Cloud Support",
+	"url": "https://support.hrcloud.com/en/help-center",
+	"email": "support@hrcloud.com"
+	}
+  },
+  "host": "corehr-api.hrcloud.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1/cloud/xApplicant": {},
```

Comment:
@HRCIntegrationsTeam Please remove empty operation, as it would not add any value to the connector